src/StorageSync/StorageSync/Models/PSRegisteredServer.cs

```diff
@@ -0,0 +1,61 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.StorageSync.Models
+{
+    public class PSRegisteredServer : PSResourceBase
+    {
+        public string StorageSyncServiceName { get; internal set; }
+        public string ServerName { get; internal set; }
+
```

Comment:
Does your team has resharper preferences. It would easier to patch Azure PS preferences, rather than fixing each instances

---
In reply to: [256594306](https://github.com/Azure/azure-powershell/pull/8527#discussion_r256594306) [](ancestors = 256594306)